x-pack/plugins/xpack_main/server/lib/__tests__/setup_xpack_main.js

```diff
@@ -58,7 +58,7 @@ describe('setupXPackMain()', () => {
 
     sinon.assert.calledWithExactly(mockServer.expose, 'info', sinon.match.instanceOf(XPackInfo));
     sinon.assert.calledWithExactly(mockServer.ext, 'onPreResponse', sinon.match.func);
-    sinon.assert.calledWithExactly(mockElasticsearchPlugin.status.on, 'change', sinon.match.typeOf('asyncfunction'));
```

Comment:
__note:__ we shouldn't rely on `asyncfunction` type, any function that returns `Promise` is `async`.